src/site/sv/index.html

```
@@ -1,13 +1,19 @@
-<html lang="sv"><head></head><body>﻿
+<html lang="sv">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)